Doc/c-api/exceptions.rst

```diff
@@ -1091,6 +1092,8 @@ the variables:
 +------------------------------------------+---------------------------------+----------+
 | :c:data:`PyExc_BytesWarning`             | :exc:`BytesWarning`             |          |
 +------------------------------------------+---------------------------------+----------+
+| :c:data:`PyExc_Py2xWarning`             | :exc:`Py2xWarning`             |          |
```

Comment:
I think the alignment of this row has gone a bit off and a bit more whitespace needs to be added?